### DISCUSSION TITLE
Announcement: @photovoltex Appointed as New Maintainer

### DISCUSSION THREAD

**Discussion by roderickvd** [librespot-org/librespot#1421] [General]
We’re excited to welcome @photovoltex to the librespot team as a maintainer!

In recent months, he has made a standout contribution with the implementation of the new `dealer` interface, which replaces the legacy `spirc` one. This update marks a major step forward in modernizing librespot’s architecture and has brought renewed momentum to the project. We also want to extend our thanks to everyone who has supported this work through code reviews, bug reports, and feedback—your input has been invaluable in driving librespot forward.

The existing maintainers will continue in their roles, ensuring that librespot stays on course as a team effort.

Let’s build on this momentum! Please join us in supporting @photovoltex in his new role and continuing to contribute to make librespot even better. 🥂

**Comment by kingosticks** [comment]
Excellent news. Welcome @photovoltex.

And huge thanks to @roderickvd for all the hard work he's put in.

**Comment by photovoltex** [comment]
Thanks for having me^^ and the already done work on this project.

I'm new to this kind of position/work, but will try to help and support were I can :)
